src/libraries/System.Drawing.Common/src/System/Drawing/Text/PrivateFontCollection.cs

```diff
@@ -70,6 +71,7 @@ public void AddFontFile(string filename)
                 // This is the default behavior on Desktop. The name "path" originates from Path.GetFullPath or similar which would refuse
                 // a null value.
                 throw new ArgumentNullException("path");
+#pragma warning restore CA2208
```

Comment:
Nit: straddling a pragram disable/restore across partial if blocks like this is odd.  Even though it's more pragmas, I'd prefer if we surrounded each throw tightly with its own set of pragmas in a case like this.